larray_editor/view.py

```diff
@@ -554,7 +553,7 @@ def paste(self):
 
     def plot(self):
         if not matplotlib_present:
-            raise Exception("plot() is not available because matplotlib is not installed")
+            QMessageBox.critical(self, "Error", "plot() is not available because matplotlib is not installed")
```

Comment:
I guess nobody will ever trigger this but it's not too much code so let's keep this.